boards/frdm-k22f/Makefile.dep

```diff
@@ -3,4 +3,4 @@ ifneq (,$(filter saul_default,$(USEMODULE)))
   USEMODULE += saul_adc
 endif
 
--include $(RIOTCPU)/k22f/Makefile.dep
+include $(RIOTCPU)/$(CPU)/Makefile.dep
```

Comment:
why use `$(CPU)` here instead of `kinetis` as in all other places, e.g., `Makefile.features`

Reply:
I mean, why not use $(CPU) when it is available? The reason for hard coding the name in Makefile.features is that it is loaded before Makefile.include in the board directory, so CPU has not been assigned a value yet.